spec/editor-linter-spec.coffee

```diff
@@ -0,0 +1,41 @@
+describe 'editor-linter', ->
+  EditorLinter = require('../lib/editor-linter')
+  beforeEach ->
+    waitsForPromise ->
+      atom.workspace.destroyActivePaneItem()
+      atom.workspace.open('test.txt')
+  describe '::constructor', ->
```

Comment:
whitespace is your necessary for readablilty. separate describes and tests with empty lines. separate setup from expectation with whitespace.